source/gui/configProfiles.py

```diff
@@ -29,6 +31,8 @@ def __init__(self, parent):
 		ProfilesDialog._instance = self
 		# Translators: The title of the Configuration Profiles dialog.
 		super(ProfilesDialog, self).__init__(parent, title=_("Configuration Profiles"))
+		self.helpIds[self.GetId()] = "ConfigurationProfiles"

```

Comment:
Just a thought: are these translated?

Reply:
The anchors in the user guide are not translated, but the help text which loads should be in the users language.